tests/pytests/test_scorers.py

```diff
@@ -291,3 +291,19 @@ def testScoreError(env):
     waitForIndex(env, 'idx')
     env.expect('ft.add idx doc1 0.01 fields title hello').ok()
     env.expect('ft.search idx hello EXPLAINSCORE').error().contains('EXPLAINSCORE must be accompanied with WITHSCORES')
+
+def testExposeScore(env: Env):
+    env.expect('FT.CREATE idx ON HASH SCHEMA title TEXT').ok()
+    with env.getClusterConnectionIfNeeded() as conn:
+        conn.execute_command('HSET', 'doc1', 'title', 'hello')
+        conn.execute_command('HSET', 'doc2', 'title', 'world')
+
+    doc1_score = 1 if env.isCluster() else 2 # TODO: why?
```

Comment:
Isn't it because the score is calculated with respect to the local shard only? (hence different when documents are spread)

Reply:
I think so. I can remove the comment

Reply:
or - have a comment that explains the reason for the difference :)